collect-filling-forms.rst

```diff
@@ -0,0 +1,220 @@
+
+   .. image:: /img/collect-completing-forms/general-settings-highlight-user-interface.* 
+     :alt: The General Settings menu of the Collect app. The *User Interface* item is circled in red.
+  
+4. Select :menuselection:`Navigation`
+
+   .. image:: /img/collect-completing-forms/user-interface-highlight-navigation.* 
+     :alt: The User Interface menu of the Collect app. The *Navigation* item is circled in red.
+
+5. Update your form navigation preference  
+
+   .. image:: /img/collect-completing-forms/ui-navigation-buttons.* 
+     :alt: The User Interface menu of the Collect app, as displayed in the previous image. There is now a modal titled *Navigation*, with radio buttons (single select) for: *Use horizontal swipes*, *Use forward/back buttons*, and *Use swipes and buttons*. The option for *Use forward/back buttons* is selected and circled in red.
+  
+.. image:: /img/collect-completing-forms/question-screen-with-buttons.* 
+  :alt: A question screen in the Collect App. There are now two buttons below the question text, with left (backwards) and right (forwards) buttons.
+ 
+
+Jumping to questions
+~~~~~~~~~~~~~~~~~~~~~~
+  
+The arrow icon (|arrow|) in the top right corner opens the jump menu. From the jump menu, you can go to any question or go to the beginning/ending of the form.
+
+.. |arrow| image:: /img/collect-forms/jumpicon.*
+             :alt: Opens the jump menu. 
+
+ 
+.. image:: /img/collect-forms/jumpscreen.*
+    :alt: Screen with the arrow icon displayed in ODK Collect on an Android phone. 
+    :class: device-screen-vertical
+  
+.. image:: /img/collect-forms/jumpmenu.*
+    :alt: Jump menu displayed in ODK Collect on an Android phone. 
+    :class: device-screen-vertical
+ 
+.. note::
+
+  If a form contains questions in a looped group, those questions will only appear in the Jump menu once an actual record is created.
+ 
+Autoadvance Questions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some questions will :ref:`auto-advance <autoadvance-widget>` after being answered.
+
+.. raw:: html
+
+  <video controls muted style="max-width:100%">
+    <source src="/form-widgets/auto-advance.mp4">
+  </video>
+
+
+Required Questions
+~~~~~~~~~~~~~~~~~~~~~~
+
+`Required questions <http://xlsform.org/#required>`_ will not allow you to advance unless answered.
+
+.. image:: /img/form-widgets/trigger-sorry.*
+  :alt: A question screen in the Collect app. An error text reads, "Sorry, this response is required."
+
+
+.. _completing-form:
+
+Completing a Form
+-------------------
+
+Once you have reached the end of a form, you will have the opportunity to *Save* and *Exit* the form. 
+
+
+.. image:: /img/collect-completing-forms/save-and-exit.* 
+  :alt: The end of a survey in the Collect app. The headline is *You are at the end of Demo Survey.* Below that is a text field labeled *Name this form*, with the value 'Demo Survey'. Then an unchecked checkbox labeled *Mark form as finalized*. Below all that is a button labeled *Save Form and Exit*.
```

Comment:
Inconsistent use of labeled and labelled. We should go with **labeled** according to the style guide.